src/pages/gateway/cache-control-headers.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Edge caching
+description: Specifies how to use headers to limit and modify the cache for GET requests.
+---
+
+# Edge caching
+
+API Mesh for Adobe App Builder supports edge caching for users who provide their own content delivery network (CDN), such as Fastly. Edge caching allows you to improve website load times and reduce consumption costs associated with bandwidth.
+
+## Cache-control headers
+
+CDN's use cache-control headers to determine how your information is cached.
+
+Every time you use a browser or a GET request to access a url, the site's response headers typically include a `cache-control` header, which determines how long the site will allow its data to be cached. For example, a website could have the following response header:
+
+```html
+cache-control: max-age=0
+```
+
+The `max-age=0` cache-control directive means that this site wants to serve [fresh](https://developer.mozilla.org/en-US/docs/Web/HTTP/Caching#fresh_and_stale_based_on_age) data until `0` seconds after the response is generated, which basically means it never wants to cache data. This could also be achieved with a `no-store` directive.
+
+For more information on specific cache-control directives and how they can be used, see the [Mozilla Developer Network's cache control guide](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control).
+
+## Using cache-control headers with API Mesh
+
+API mesh users can add cache-control headers to [request headers](#as-request-headers) or to a [mesh configuration file](#in-the-mesh-configuration-file).
+
+<InlineAlert variant="info" slots="text"/>
+
+Currently query-level caching is not supported.
+
+### As request headers
+
+When making a GET request, you can add cache-control headers to your GraphQL client.
```

Comment:
I think of the GraphQL client as a tool such as GraphiQL.  Technically, the cache-control headers are provided from the individual sources that make up the mesh config file.